sharkfuser/include/fusilli/attributes/tensor_attributes.h

```diff
@@ -153,6 +154,34 @@ inline std::vector<size_t> getChannelsLastStrideOrder(size_t numDims) {
   return strideOrder;
 }
 
+// Generates a stride order preserving the format of the input tensor. For
+// example an input of {10, 30, 20} would return a stride order of {0, 2, 1}.
```

Comment:
This docstring may not be in sync with the signature which takes an `inputStride` and `outputDimSize`.